skycoin/skywire
# Remove heartbeat | Add Deregister

Did you run `make format && make check`? Yes

Fixes [Skywire Service #447](https://github.com/SkycoinPro/skywire-services/issues/447)

 Changes:	
- remove updater loop
- change `Delete` entry to `Deregister`
- add `Deregister` on client side (skywire) stop app process and visor shutdown process.

How to test this PR:
- Run visor with vpn-server true, then stop it by hypervisor UI or shutdown visor from CLI. Then check [`sd.skycoin.com/api/services?type=vpn`](sd.skycoin.com/api/services?type=vpn) for related PK.
